src/lib.rs

```diff
@@ -148,9 +152,11 @@ where
     /// Sets display [Orientation]
     ///
     pub fn set_orientation(&mut self, orientation: Orientation) -> Result<(), Error<RST::Error>> {
+        let value = (self.madctl & 0b0001_1111) | (orientation as u8);
```

Comment:
Am wondering if there is anything we should do to make it crystal clear that the orientation value should never be setting the lower 5 bits, Maybe: `orientation & 0b1110_0000` or `orientation << 5`?

Reply:
good point